tenable/io/v3/was/__init__.py

```diff
@@ -17,11 +17,15 @@
     :glob:
 
     folders
+    templates
+    user-templates
     scans
```

Comment:
Please follow the ASCII order. So, Add agent_groups after agent_config, similar to the order in imports.